Data_cleaning/load_clean_concat.ipynb

```diff
@@ -200,15 +295,36 @@
     "    for file_name in file_names:\n",
     "        df = pd.read_csv(data_folder+file_name+'.csv', parse_dates=['TIMESTAMP'])\n",
     "        df.rename(mapper=remove_pit_suffix, axis='columns', inplace=True)\n",
-    "            \n",
+    "        \n",
     "        if print_text_result: print(f'===== {file_name} =====')\n",
+    "        \n",
+    "        diff_dict = dict()\n",
+    "        prune_unnecessary_features(df)\n",
+    "        \n",
+    "        diff_dict['dates_removed'] = filter_dates(df, file_name)\n",
+    "        diff_dict['duplicates_removed'] = filter_duplicate_date(df)\n",
+    "        \n",
+    "        # here is where the wavelet coefficients are calculated \n",
+    "        df = fill_na_redox_values(df, file_name)\n",
```

Comment:
Im not sure if this function should be used (fill_na_redox_values):
	Filling missing redox values might cause erroneous redox values treated as normal of vice verca (these can depend on values on other columns in same row).

Should we use filter_missing_values instead of this function?

Reply:
there were just one or two missing values when I was developing wavelet feature module if I remember correctly so I thought it's better instead of either skewing wavelet calculations or removing a lot of data to fill it in instead

Reply:
I agree on this decision. Its also better for training the model, and few fixed rows don't hurt training as much as removing more rows :+1: